content/en/docs/reference/resource-types/_index.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Resource Types
+---
+
+<!-- overview -->
+
+This section provides information on the different resource types you can request, built into Kubernetes.
+
+<!-- body -->
+
+There are 3 different resource types: `cpu`, `memory`, and `storage`.
+
```

Comment:
I recommend telling the reader that resources are relevant to [scheduling](https://kubernetes.io/docs/concepts/scheduling-eviction/) as well as for enforcing limits.

Reply:
Sounds good!